errutil.go

```diff
@@ -119,6 +140,17 @@ func Caller(skip int) Frame {
 	return s
 }
 
+// Callers(0) returns the frames starting from the caller of Caller.
+func Callers(skip int) []Frame {
+	var pcs [64]uintptr
+	n := runtime.Callers(skip+2, pcs[:]) // +2 since Callers gives here passing 1, not 0.
+	var frames []Frame
+	for i := 0; i < n; i++ {
```

Comment:
I wonder if this would give different/better results calling CallersFrames on it, and then getting a slice of locations that are used for a newFrameError2  instead of giving it the raw Frame 🤔 The contract here at least is small so we can change the internals if we find that out. Perhaps unexport `Callers()` though, since we currently only need WithStack.

Reply:
Unexported callers at least.